roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -13,6 +13,19 @@
     additive_facts_to_overwrite:
     - "{{ 'master.named_certificates' if overwrite_named_certs | bool else omit }}"
 
+- stat:
+    path: "{{ named_certs_dir }}"
+    get_checksum: false
+    get_attributes: false
+    get_mime: false
+  register: named_certs_dir_stat
+
+- name: Backup Named Certificates
+  command: >
+    tar -czvf /etc/origin/master-named-certificates-backup-{{ ansible_date_time.epoch }}.tgz
+    {{ named_certs_dir }}
+  when: named_certs_dir_stat.stat.exists
+
```

Comment:
Added backup option for named certificates.

The named certs are in fact backup by the certificate-backup.yml playbook as part of the /etc/origin/master directory
https://github.com/openshift/openshift-ansible/blob/release-3.11/playbooks/openshift-master/private/certificates-backup.yml#L37-L40